kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -203,19 +210,8 @@ var TextView = BaseView.extend({
         return this;
     },
 
-    text_changed: function() {
-        this.model.set(this.text_attr, this.$(".rating-text-feedback")[0].value);
-    },
-
-    toggle_disabled: function(val) {
-        if( typeof val === "undefined" ) {
-            val = !this.is_disabled();
-        }
-        this.$(".rating-text-feedback").attr("disabled", val);
-        return this;
-    },
-
-    is_disabled: function() {
-        return this.$(".rating-text-feedback").attr("disabled") === "disabled";
-    }
+    text_changed: _.throttle(function() {

```

Comment:
What is probably better to do, is not to debounce the set, but just debounce the save.

Then when the 'close' method is called on the contentareaview, you can call a non-debounced save to ensure no user data is lost.

This is what happens in point updating also.